docs/core/docker/publish-as-container.md

```diff
@@ -182,6 +182,24 @@ You can control many aspects of the generated container through MSBuild properti
 > [!NOTE]
 > The only exceptions to this are `RUN` commands. Due to the way containers are built, those cannot be emulated. If you need this functionality, you'll need to use a _Dockerfile_ to build your container images.
 
+:::zone pivot="dotnet-8-0"
+
+### `ContainerArchiveOutputPath`
+
+Starting in .NET 8, you can create a container directly as a _tar.gz_ archive. This feature is useful if your workflow isn't straightforward and requires that you, for example, run a scanning tool over your images before pushing them. Once the archive is created, you can move it, scan it, or load it into a local Docker toolchain.
+
+To publish to an archive, add the `ContainerArchiveOutputPath` property to your `dotnet publish` command, for example:
+
+```dotnetcli
+dotnet publish \
+  -p PublishProfile=DefaultContainer \
+  -p ContainerArchiveOutputPath=./images/sdk-container-demo.tar.gz
+```
+
+You can specify either a folder name or a path with a specific file name.
```

Comment:
A bit of clarification for this point - if they specify a folder name only, the file name generated for the image archive file will be $(ContainerRepository).tar.gz. these archives can containe multiple tags inside them so only one file is created for all ContainerImageTags.